modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/model/DiscoverModel.kt

```diff
@@ -64,6 +64,7 @@ data class DiscoverRow(
     @field:Json(name = "sponsored") val sponsored: Boolean = false,
     @field:Json(name = "curated") override val curated: Boolean = false,
     @field:Json(name = "sponsored_podcasts") val sponsoredPodcasts: List<SponsoredPodcast> = emptyList(),
+    val regionCode: String? = null,
```

Comment:
I would like to refactor the way we display items on Discover view. Ideally we should create a model for UI instead of using the server model. I decided to not address this refactor on this PR since I would need to modify others part of the app

Reply:
Hah, I asked about this in a comment I deleted. 😅 I think it is fine to keep it for now if you want to refactor it later because we already use this pattern for some other classes.